spec.html

```diff
@@ -3720,6 +3731,8 @@
           </tbody>
         </table>
       </emu-table>
+      <!-- If ToObject is ever made hookable, we need to alert the editors of the HTML Standard as
+           that'll effect the security model of the Location exotic object. -->
```

Comment:
I think it's important. New hooks in ECMAScript have resulted in security issues in browsers at least once (`@@toPrimitive` and `Location`) and also tend to cause breakage due to "cross-origin objects" throwing on unknown property access (the way to fix this is by adding e.g., `@@toPrimitive` with value undefined on WindowProxy).

Happy to do whatever you think is best here.

I'll fix the issues @UltCombo pointed out once we figure out a plan.